tutorials/plot_stats_cluster_1samp_test_time_frequency.py

```diff
@@ -84,8 +84,9 @@
 # Compute statistic
 # -----------------
 threshold = 2.5
+n_permutations = 100  # Warning: 100 is too small for real-world analysis.
```

Comment:
Good enough for me. If you want to get fancy you can put this in the `###` block above with a `.. warning:: ` so it gets a special angry-looking box around it